src/Integration.Vsix/CFamily/CFamilyIssueToAnalysisIssueConverter.cs

```diff
@@ -109,9 +116,17 @@ public IAnalysisIssue Convert(Message cFamilyIssue, string sqLanguage, ICFamilyR
             Debug.Assert(cFamilyIssue.Column > 0 || cFamilyIssue.Column == 0);
             Debug.Assert(cFamilyIssue.EndColumn > 0 || cFamilyIssue.EndLine == 0);
 
+            var ruleMetaData = rulesConfiguration.RulesMetadata[cFamilyIssue.RuleKey];
```

Comment:
is it possible here to get a key that's not present in the dictionary? I would leave a comment or use TryGet

Reply:
No, it's not possible.